infrastructure/quick-deploy/aws/all/parameters.tfvars

```diff
@@ -458,19 +458,20 @@ ingress = {
 
 extra_conf = {
   core = {
-    Amqp__AllowHostMismatch                    = false
-    Amqp__MaxPriority                          = "10"
-    Amqp__MaxRetries                           = "5"
-    Amqp__QueueStorage__LockRefreshPeriodicity = "00:00:45"
-    Amqp__QueueStorage__PollPeriodicity        = "00:00:10"
-    Amqp__QueueStorage__LockRefreshExtension   = "00:02:00"
-    MongoDB__TableStorage__PollingDelayMin     = "00:00:01"
-    MongoDB__TableStorage__PollingDelayMax     = "00:00:10"
-    MongoDB__TableStorage__PollingDelay        = "00:00:01"
-    MongoDB__DataRetention                     = "10.00:00:00"
-    MongoDB__AllowInsecureTls                  = true
-    Redis__Timeout                             = 3000
-    Redis__SslHost                             = ""
+    Components__QueueAdaptorSettings__AdapterAbsolutePath = "/adapters/queue/amqp/ArmoniK.Core.Adapters.Amqp.dll"
```

Comment:
This should come from the same place than the class name

Reply:
Moreover, if the adapter used is rabbitmq, the path should be `/adapters/queue/rabbit/ArmoniK.Core.Adapters.RabbitMQ.dll`

Reply:
Done